app/config/parameters.yml.dist

```diff
@@ -30,5 +30,5 @@ parameters:
     slack_webhook_url: YOU_SLACK_WEBHOOK_URL
 
     # Elastic Search Configuration
-    elastic_host: ELASTICSEARCH_HOST
-    elastic_port: ELASTICSEARCH_PORT
+    elastic_host: localhost
```

Comment:
travis uses this file for the configuration and failed because it couldn't resolve host `ELASTICSEARCH_HOST`. 
I tried to update dynamically the conf in the .travis.yml but it does not work.

IMO, it's not a problem to add by default `localhost` but if you don't want it, I can find an other way